geobench-downloader.py

```diff
@@ -0,0 +1,113 @@
+"""
+Download the GEO-Bench benchmark from Zenodo
+
+Before running the script, make sure to install the following packages:
+
+```
+pip install requests
+pip install sickle
+pip install tqdm
+```
+
+"""
+import argparse
+import os
+
+try:
+    import requests
+except:
+    print("Please install the requests package and try again: 'pip install requests'.")
+
+try:
+    from sickle import Sickle
+except:
+    print("Please install the sickle package and try again: 'pip install sickle'.")
+
+try:
+    from tqdm import tqdm
+except:
+    print("Please install the tqdm package and try again: 'pip install tqdm'.")
+
+
+COMMUNITY = "geo-bench"
+
+
+def download_zenodo_record(record_data, output_dir):
+    """
+    Download the files from a Zenodo record
+
+    Parameters
+    ----------
+    record_data : dict
+        Dictionary containing the record data (retrieved from the Zenodo API)
+    output_dir : str
+        Output directory to save the files
+
+    Returns
+    -------
+    None
+
+    """
+    # Extract file information
+    files = record_data["files"]
+
+    # Download each file
+    for file in files:
```

Comment:
I made test in the past and zenodo is throttled and slow for download. But if we launch all files in e.g. 10 threads, it would be 10x faster.

Reply:
Ok. A Simple idea is to put the download code in a function and then use multiprocessing to parallelize it. We can have ncpu in the script args.